apps/openmw/mwgui/charactercreation.cpp

```diff
@@ -105,23 +109,36 @@ namespace MWGui
         mGenerateClassSpecializations[0] = 0;
         mGenerateClassSpecializations[1] = 0;
         mGenerateClassSpecializations[2] = 0;
+
+        // Setup player stats
+        for (int i = 0; i < ESM::Attribute::Length; ++i)
+        {
+            mPlayerAttributes.insert(std::make_pair(ESM::Attribute::sAttributeIds[i], MWMechanics::AttributeValue()));
```

Comment:
You may want to use emplace instead of insert(std::make_pair()) while you're at it.